src/app/mail/mail-sidebar/compose-mail/compose-mail.component.ts

```diff
@@ -342,18 +366,18 @@ export class ComposeMailComponent implements OnInit, AfterViewInit, OnDestroy {
         }, 300);
       }
     }
+  }
 
-    if (this.forwardAttachmentsMessageId) {
-      if (this.editor) {
-        this.updateEmail();
-      } else {
-        setTimeout(() => {
-          this.updateEmail();
-        }, 1000);
-      }
+  toggleHtmleditor(value: boolean) {
```

Comment:
rename `toggleHtmleditor`  to `toggleHtmlEditor`

Reply:
why the function name is toggle if you are passing the value?
the function should rather say `setHtmlEditor`

Reply:
yes, right let me update, thanks